src/tests/roc_fec/target_openfec/test_encoder_decoder.cpp

```diff
@@ -182,5 +191,10 @@ TEST(encoder_decoder, load_test) {
     }
 }
 
+TEST(encoder_decoder, max_source_block) {
+    check_max_source_block(ReedSolomon8m, OF_REED_SOLOMON_MAX_NB_SOURCE_SYMBOLS_DEFAULT);
+    check_max_source_block(LDPCStaircase, OF_LDPC_STAIRCASE_MAX_NB_SOURCE_SYMBOLS_DEFAULT);
```

Comment:
Other encoder-decoder tests use a cycle over all codecs:

```
for (int type = ReedSolomon8m; type != CodecTypeMax; ++type) {
```

It's better to use it here as well. And thus we can also remove (inline) the check_max_source_block() method.

Reply:
Done